pandas/_libs/tslibs/period.pyx

```diff
@@ -1246,6 +1246,37 @@ cdef class _Period(object):
 
     @property
     def dayofweek(self):
+        """
+        Return the day of the week.
+
+        This attribute returns the day of the week on which the particular
+        date occurs with Monady=0, Sunday=6.
+
+        Returns
+        -------
+        Int:Range of 0 to 6
+
+        See also
+        --------
+        Period.dayofyear
+            Return the day of year.
+        Period.daysinmonth
+            Return the days in that month.
+
+        Examples
+        --------
+        >>> p=pd.Period('2012-1-1 19:00', freq='H')
```

Comment:
1) Add spaces around `p` to follow flake8 also in docstrings.
2) Perhaps better to use `period` for the variable name, it's short enough.